src/tasks/tasks.ts

```diff
@@ -211,7 +210,7 @@ export class Tasks {
             throw new Error(ERRORS.API.PAYMENT_INFO_NOT_FOUND.message)
         }
         const partialFee = paymentInfo?.partialFee
-        let response: CaptchaSolutionResponse[] = []
+        let response: DappUserSolutionResult = { captchas: [], partialFee: '0' };
```

Comment:
if fee is going to be a number, can we use a number type and cast it during transport as needed?